api/solarperformanceinsight_api/models.py

```diff
@@ -548,3 +552,371 @@ class UserInfo(StoredObject):
     """Information about the current user"""
 
     auth0_id: str = Field(..., description="User ID from Auth 0")
+
+
+class JobTimeindex(BaseModel):
+    """Parameters for a time index that all data uploads must conform to.
+    Data is assumed to time-averaged and closed and labeled at the left endpoint, i.e.
+    a datapoint at 23:00 of data with a 1 hour time step is assumed be the
+    average of data from 23:00 to 23:59.
+    """
+
+    start: dt.datetime = Field(
+        ..., description="Start of the time range that data will be uploaded for"
```

Comment:
I would add an example or two of acceptable formats.

Reply:
I've updated the description to describe the formats. Most users will likely interact through the API docs which render an example like below
![image](https://user-images.githubusercontent.com/4625457/102928597-37dc6900-4456-11eb-9381-9b67842fc604.png)

Reply:
and if you want to click through to view the schema
![image](https://user-images.githubusercontent.com/4625457/102928779-96a1e280-4456-11eb-9c8f-79ab2f293015.png)